getMarks/getMarks.js

```diff
@@ -32,35 +30,17 @@ function getMarksBySubjectId(id) {
   };
 
   if (id > 0 && id <= constants.CONSTANTS.MOCK_SUBJECTS.length) {
+    const marks = constants.CONSTANTS.MOCK_MARKS[id - 1].Marks;
+    const average = marks.reduce((prev, curr) => prev + curr) / marks.length;
```

Comment:
You have excellent Average(ID)  method. Follow the SOLID principal - use it here, please.

Reply:
it's better to transmit marks as a parameter, not id. Because this method is used in different endpoints